src/main/java/seedu/address/model/account/Name.java

```diff
@@ -26,9 +26,13 @@ public static boolean isValidName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    public String getName() {
+        return accountName;
```

Comment:
Maybe here can call the toString method like "return toString()" , or u can call getName() in toString() too to avoid duplication of code